_posts/2024-06-20-pyconus-pyopensci-sprints.md

```diff
@@ -0,0 +1,358 @@
+title: "pyOpenSci beginner-friendly sprints at PyCon US 2024"
+excerpt: "pyOpenSci holds beginner-friendly sprints with the goal of making new contributors feel welcome. Learn about our last sprint at PyCon US 2024 and tips we use to make sprints accessible to more people."
+author: "Leah Wasser"
+permalink: /blog/pyopensci-pyconus-2024-sprints.html
+header:
+  overlay_image: images/blog/2024/may/23-05-2024-get-involved.png
+  overlay_filter: rgba(20, 13, 36, 0.8)
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+
+## pyOpenSci's approach to beginner-friendly sprints
+
+In previous posts, I talked about:
+
+* Our [incredible experience at PyCon US 2024](recap-pyos-pyconus-2024.html).
+* My [personal experiences with the Python packaging ecosystem and building consensus
+  within the scientific Python community](python-packaging-friends-dont-let-friends-package-alone.html).
+
+Here, I will share with you what we have learned at pyOpenSci through holding
+beginner-friendly sprints over the past 2 years.
+Specifically, I want to
+explore the varied motivations and barriers associated with contributions to
+Open Source, and how pyOpenSci is addressing them.
+
+<div class="notice--success" markdown="1">
+## TL;DR
+
+* I believe that we can get more people involved in Open Source if it's done the right
+  way. But despite the word "open" in the name, Open Source is not necessarily open to
+  all people.
+* By setting up infrastructure such as project boards and tagging issues as beginner-
+  friendly, you are on your way towards a beginner-friendly sprint.
+* Be prepared to support people who are newer to Git and GitHub.
+* Create opportunities for beginners to contribute to documentation. The beginner lense
+reviewing your docs is critical to
+  ensuring the documentation is usable! Documentation contributions can also be
+  implemented completely on GitHub without needing to use the command line.
+</div>
+
+## pyOpenSci's PyCon US 2024 Sprint
+
+I've been running pyOpenSci sprints for about 2 years now. This year was my second year
+leading a PyCon US sprint for pyOpenSci. I've learned a lot! While last year we had 2
+sprinters, this year we had a tremendous turnout of 18 people from several countries
+and across the United States for our 1-day sprint. This resulted in over 35 issues and
+pull requests where volunteers helped fix issues on our website, in our tutorials, and
+in our packaging and peer review guidebooks. Many long-standing issues and bugs were
+fixed thanks to this wonderful Python community.
+
+We had so much activity that I'm still working on closing up issues and pull
+requests now—weeks after the sprint
+ended!
+
+After such a tremendous experience this year, I wanted to share with you the things
+that we've learned at pyOpenSci about running beginner-friendly sprints.
+
+## What is a sprint like?
+
+If you haven’t been to a sprint before, it’s an experience every open source
```

Comment:
open source was capitalized in earlier parts of the post, but here it appears lower case. it should be consistent throughout the post.